docs: Add Tx Service data decoder guide (#489)
safe-global/safe-docs: pages/core-api/api-safe-transaction-service.mdx

- [**Etherscan** configured networks](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/clients/etherscan_client.py#L24)
- [**Blockscout** configured networks](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/clients/blockscout_client.py#L21)

> The following [Guides](../core-api/transaction-service-guides/data-decoder.mdx) show how to decode contract interactions data of a transaction from the Safe Transaction Service.

Reviewer: ```suggestion
Follow our [guides](https://add_data_decoder_guide--docs.review.5afe.dev/core-api/transaction-service-guides/data-decoder) to learn how to decode contract interaction data using the Safe Transaction Service for a transaction.
```

Author: Updated here https://github.com/safe-global/safe-docs/pull/489/commits/c17918ee6926fd038220681598adada7cefdff80 because the link is incorrect. Maybe we can do the same update on lines 154 and 126

Reviewer: Duh, my bad @falvaradorodriguez! Makes sense on the same update to the lines you mentioned.

Author: Done here https://github.com/safe-global/safe-docs/pull/489/commits/5bf8839ea59c4d7e796a356e68b4a2445e3c0ae6. Thanks @tanay1337
